app/components/forms/user-payment-info-form.js

```diff
@@ -88,19 +88,24 @@ export default Component.extend(FormMixin, {
         }
       }
     };
-  },
-  actions: {
-    submit() {
-      this.onValid(async() => {
-        try {
-          this.authManager.currentUser.setProperties(this.userBillingInfo);
-          await this.authManager.currentUser.save();
-          this.notify.success(this.l10n.t('Your billing details has been updated'));
-        } catch (error) {
-          this.authManager.currentUser.rollbackAttributes();
-          this.notify.error(this.l10n.t('An unexpected error occurred'));
-        }
-      });
-    }
   }
-});
+
+  @computed
```

Comment:
Add a bracket to it (only applies to computed) ... Needs to be a function. Has a buggy behaviour other wise sometimes when used with certain addons.

Reply:
@niranjan94  Oh! okay. doing so. Although this one doesn't require any property to be watched, right?

Reply:
@niranjan94 Done!